server/sonar-web/src/main/js/apps/coding-rules/facets/status-facet.js

```diff
@@ -26,15 +26,15 @@ export default BaseFacet.extend({
 
   getValues () {
     const values = this.model.getValues();
-    const x = values.map(function (value) {
+    const x = values.map(value => {
       return _.extend(value, { label: translate('rules.status', value.val.toLowerCase()) });
     });
     return x;
   },
 
   sortValues (values) {
     const order = this.statuses;
-    return _.sortBy(values, function (v) {
+    return _.sortBy(values, v => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)